doc/src/modules/solvers/solveset.rst

```diff
@@ -398,7 +398,7 @@ How does ``solveset`` ensure that it is not returning any wrong solution?
     >>> from sympy import symbols, S, pprint, solveset
     >>> x, n = symbols('x, n')
     >>> pprint(solveset(abs(x) - n, x, domain=S.Reals), use_unicode=True)
-    [0, ∞) ∩ {n}
+    {x | x ∊ {-n, n} ∧ ({{n} ∈ [0, ∞)})}
```

Comment:
I have updated the conditions to be a boolean predicate instead of being a set.
I would like to know which output is preferred so as to modify the set of conditioned symbols( or expressions).

Ping @aktech @asmeurer @hargup @smichr